### PR TITLE
Add Unjail command

### PR DESCRIPTION
**Pull request by alpe**
Adds: `$ tgrade tx poe unjail --from mykey`
+ contract integration test case

### CODE REVIEW

File: x/poe/client/cli/tx.go [hunk 1]
```diff
@@ -394,6 +398,49 @@ $ %s tx poe unbond 100stake --from mykey
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
 
+func NewUnjailTxCmd() *cobra.Command {
```
Comment (ethanfrey) [inline x/poe/client/cli/tx.go:404]: But this is just for tests, right? You mention below in a comment that only the OC can do this.

If it is for actual (non-test) usage, we should make an OC proposal command. But that may be overkill with the UI already working

**Review by ethanfrey — COMMENTED** [review]
Some thoughts

**Review by ethanfrey — APPROVED** [review]
You can merge as-is if you want. I think some of the type parsing could be cleaner, but that is the Rust side.

I made an issue: https://github.com/confio/poe-contracts/issues/20 Let's revisit this sometime before 0.6 but not now.

File: x/poe/client/cli/tx.go (hunk 1 above)
Comment (alpe) [reply x/poe/client/cli/tx.go:404]: The node operator can unjail his validator after the jail lock expired. The OC can unjail any time. 
This CLI command is for the node operator only.

File: x/poe/contract/tgrade_valset.go [hunk 2]
```diff
@@ -176,11 +195,39 @@ type ValsetEpochResponse struct {
...
+	JailedUntil *JailingPeriod    `json:"jailed_until,omitempty"`
+}
+
+type JailingPeriod struct {
+	Forever bool      `json:"forever,omitempty"`
+	Until   time.Time `json:"until,omitempty"`
 }
 
+func (j *JailingPeriod) UnmarshalJSON(data []byte) error {
+	var r struct {
```
[the remark this answers is not in the document]
Comment (alpe) [reply x/poe/contract/tgrade_valset.go:210]: Not sure if I understand this correct. The `Until` field contains the unix timestamp as string. To hide this from the application logic, I convert this into a proper Go time object.

The cases to cover:
* Until timestamp: "jailed_until":{"Until":"1587556801000000000"}}}
* Not jailed: "jailed_until":null}}
* Jailed forever: "jailed_until":{"Forever":{}}}}

**Pull request merged**